src/Bpost/Order/Box.php

```diff
@@ -264,7 +264,6 @@ public static function createFromXML(\SimpleXMLElement $xml)
             $className = '\\Bpost\\BpostApiClient\\Bpost\\Order\\Box\\' . ucfirst($internationalBoxData->getName());
 
             if (!method_exists($className, 'createFromXML')) {
-                var_dump($className);
```

Comment:
We cannot remove the `var_dump` as is.  
If we remove the `var_dump`, we have to set a message in the `BpostNotImplementedException` to show which `$className` does not contain method `createFromXML`.

Reply:
I followed the pattern used at a a lot of place into code. But in fact seems missing. Added (and at others places also)